memory/spring-ai-model-chat-memory-jdbc/src/main/java/org/springframework/ai/chat/memory/jdbc/aot/hint/JdbcChatMemoryRepositoryRuntimeHints.java

```diff
@@ -35,8 +35,7 @@ public void registerHints(RuntimeHints hints, ClassLoader classLoader) {
 			.registerType(DataSource.class, hint -> hint.withMembers(MemberCategory.INVOKE_DECLARED_METHODS));
 
 		hints.resources()
-			.registerPattern("org/springframework/ai/chat/memory/jdbc/schema-mariadb.sql")
-			.registerPattern("org/springframework/ai/chat/memory/jdbc/schema-postgresql.sql");
+			.registerPattern("org/springframework/ai/chat/memory/jdbc/schema-*.sql");
```

Comment:
I believe this is the cause of the build error 

`Failed to execute goal io.spring.javaformat:spring-javaformat-maven-plugin:0.0.43:validate (default) on project spring-ai-model-chat-memory-jdbc: Formatting violations found in the following files:
Error:   * /home/runner/work/spring-ai/spring-ai/memory/spring-ai-model-chat-memory-jdbc/src/main/java/org/springframework/ai/chat/memory/jdbc/aot/hint/JdbcChatMemoryRepositoryRuntimeHints.java` 


it should be inline 

```java
	hints.resources().registerPattern("org/springframework/ai/chat/memory/jdbc/schema-*.sql");
```